request/fetch.go

```diff
@@ -0,0 +1,140 @@
+package request
+
+import (
+	"bytes"
+	goerrors "errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/browserpass/browserpass-native/errors"
+	"github.com/browserpass/browserpass-native/response"
+	log "github.com/sirupsen/logrus"
+)
+
+func fetchDecryptedContents(request request) {
+	responseData := response.MakeFetchResponse()
+
+	if !strings.HasSuffix(request.File, ".gpg") {
+		log.Errorf("The requested password file '%v' does not have the expected '.gpg' extension", request.File)
+		response.SendErrorAndExit(
+			errors.CodeInvalidPasswordFile,
+			"The requested password file does not have the expected '.gpg' extension",
+			&map[string]string{"action": "fetch", "file": request.File},
+		)
+	}
+
+	store, ok := request.Settings.Stores[request.Store]
+	if !ok {
+		log.Errorf(
+			"The password store '%v' is not present in the list of stores '%v'",
+			request.Store, request.Settings.Stores,
+		)
+		response.SendErrorAndExit(
+			errors.CodeInvalidPasswordStore,
+			"The password store is not present in the list of stores",
+			&map[string]string{"action": "fetch", "name": request.Store},
```

Comment:
Can this please be called `"store"` or `"storeName"` (or something else similar), instead of `"name"`? More descriptive :-).

Reply:
`store` -> `storeName`
`path` -> `storePath`

`file` I'll keep as `file` even thought it might sometimes be relative or full path to a file or just the file name, it depends on the error itself 😉